docs/docs/step-by-step-walkthrough.md

```diff
@@ -159,9 +171,9 @@ Push the changes to your repo:
     git push origin master
 
 
-### 3 Post github deployment:
+## 3. Post GitHub deployment:
 
-After the files are in github all CICD changes will done via github actions and triggered by a commit to master. To use gitops, make a change to the `qhub-ops.yaml` in a new branch and create pull request into master. When the pull request is merged, it will trigger a deployement of all of those changes to your QHub.
+After the files are in github all CI/CD changes will be done via github actions and triggered by a commit to master. To use gitops, make a change to the `qhub-ops.yaml` in a new branch and create pull request into master. When the pull request is merged, it will trigger a deployement of all of those changes to your QHub.
```

Comment:
```suggestion
After the files are in Github all CI/CD changes will be triggered by a commit to main and deployed via GitHub actions. To use gitops, make a change to `qhub-ops.yaml` in a new branch and create a pull request into main. When the pull request is merged, it will trigger a deployment of all of those changes to your QHub.
```